ledger-core/runner/runner.go

```diff
@@ -173,15 +182,15 @@ func (r *DefaultService) executeMethod(
 
 	classReference, err := objectDescriptor.Class()
 	if err != nil {
-		return nil, errors.W(err, "couldn't get class reference")
+		return nil, errors.W(err, "couldn't get class reference", ErrorDetail{DetailEmptyClassRef})
 	}
 	if classReference.IsEmpty() {
 		panic(throw.IllegalState())
 	}
 
 	classDescriptor, codeDescriptor, err := r.Cache.ByClassRef(ctx, classReference)
 	if err != nil {
-		return nil, errors.W(err, "couldn't get descriptors")
+		return nil, errors.W(err, "couldn't get descriptors", ErrorDetail{DetailBadClassRef})
```

Comment:
I was thinking about 
```
var  ErrBadClassRef = ErrorDetail{DetailBadClassRef}
throw.W(err, "couldn't get descriptors", ErrBadClassRef)
```
so it would be easier to use.

Reply:
imho, this is better not to do - to avoid any sharing of details among different error instances